spec/dor/publishable_spec.rb

```diff
@@ -340,6 +341,28 @@ class ItemizableItem < ActiveFedora::Base
       end
     end
 
+    context 'publish_notify_on_success' do
+      let(:dir) { '/my/dir' }
+      it 'writes empty file' do
+        Dor::Config.push! {|config| config.stacks.local_recent_changes dir}
+        expect(File).to receive(:directory?).with(dir).and_return(true)
+        expect(FileUtils).to receive(:touch).with(dir + '/ab123cd4567')
+        @item.publish_notify_on_success
+      end
+      it 'writes empty file even when given only the base id' do
+        Dor::Config.push! {|config| config.stacks.local_recent_changes dir}
+        expect(File).to receive(:directory?).with(dir).and_return(true)
+        expect(@item).to receive(:pid).and_return('aa111bb2222')
+        expect(FileUtils).to receive(:touch).with(dir + '/aa111bb2222')
+        @item.publish_notify_on_success
+      end
+      it 'raises error if not configured' do
```

Comment:
i might describe this as `misconfigured` or `not configured properly` instead of `not configured`, since it seems like this error could come up if the configuration were present but pointing to a non-existent/inaccessible dir.